packages/notification-center/src/components/notification-center-web-component/notification-center-component.tsx

```diff
@@ -21,6 +21,7 @@ export const NOTIFICATION_CENTER_PROPS = [
   'stores',
   'tabs',
   'showUserPreferences',
+  'allowedNotificationActions',
```

Comment:
please update the docs here: https://docs.novu.co/notification-center/web-component#properties